src/main/resources/db/changelog/changes/03-create-roles-table.yml

```diff
@@ -0,0 +1,21 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-roles-table
+      author: DanilKi
+      changes:
+        - createTable:
+            tableName: roles
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: name
+                  type: varchar(45)
+                  constraints:
+                    nullable: false
+                    unique: true
```

Comment:
```suggestion
                    unique: true
                    
```

Reply:
Add blank line at the end. Check other changelogs

Reply:
It is strange, because there are empty lines in every source file on my side. Have no idea, why they aren't being displayed on github.